robbot.py

```diff
@@ -26,6 +16,33 @@
 
 bot = commands.Bot(command_prefix='.')
 
+'''
+    Cog Functions & Commands
+'''
+def load_cogs():
+    for files in os.listdir('./cogs'):
+        if files.endswith('.py'):              
+            bot.load_extension(f'cogs.{files[:-3]}')
+            print(f'{files[:-3]} extension loaded')
```

Comment:
A more readable way of trimming the file extension would be to call `os.path.splitext(file)` and use that result in the load call. 

```suggestion
            cog_name, file_ext = os.path.splitext(file)
            bot.load_extension(f'cogs.{cog_name}')
            print(f'{cog_name} extension loaded')
```

Optionally, `file_ext` could be named `_`, which by convention denotes that it's unused - personally feel that `file_ext` is a bit more clear.